pages/linux/nmcli-radio.md

```diff
@@ -0,0 +1,29 @@
+# nmcli radio
+
+> Show radio switches status or enable and disable switches.
+> More information: <https://networkmanager.dev/docs/api/latest/nmcli.html>.
+> See also: `nmcli agent`.
+
+- Show status of Wi-Fi in NetworkManager:
+
+`nmcli radio wifi`
+
+- Turn Wi-Fi on|off in NetworkManager:
+
+`nmcli radio wifi {{on|off}}`
+
+- Show status of WWAN in NetworkManager:
+
+`nmcli radio wwan`
+
+- Turn WWAN on|off in NetworkManager:
```

Comment:
```suggestion
- Turn WWAN on or off in NetworkManager:
```

Reply:
I am not sure whether it's a good decision to use `or` between some "keywords" (on, off in our case). Visually it's easier to delimit them via some special symbol such as `|` or `/`. I like use pipe because it's already used in placeholders.

Reply:
I usually prefer `|` as well, but since the idea is to be clear for new command-line users, i tend to like `or` as a more explicit form on the command description. As for the command itself, i think `{{on|off}}` makes sense.

Reply:
You can choose  `bluetoothctl` or `systemctl` as reference.

Reply:
Maybe better to use `or` when alternatives are simple words and prefer `/` (for simplicity instead of pipe symbol) when alternatives consist of several words.